source/extensions/http/header_formatters/preserve_case/config.h

```diff
@@ -38,7 +38,9 @@ class PreserveCaseFormatterFactoryConfig
     : public Envoy::Http::StatefulHeaderKeyFormatterFactoryConfig {
 public:
   // Envoy::Http::StatefulHeaderKeyFormatterFactoryConfig
-  std::string name() const override { return "preserve_case"; }
+  std::string name() const override {
```

Comment:
Is there something that can systematically done to style check these names in presubmit?

Reply:
OK, just a suggestion for making this more defensible, I'll merge.